package.json

```diff
@@ -39,6 +39,10 @@
         "title": "Anki: Send To Own Deck",
         "enablement": "editorLangId == markdown"
       },
+      {
+        "command": "anki.sendDir",
+        "title": "Anki: Send All"
```

Comment:
I'm not sure "Send All" is clear enough. Maybe "Send all in directory"? Also is it every card in the `${workspaceFolder}` or the current directory the card is in?

Reply:
It's every md file in the workspace. It recursively searches all subdirectories.

I agree the name isn't clear